workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/WorkflowFragment.kt

```diff
@@ -78,14 +78,9 @@ abstract class WorkflowFragment<PropsT, OutputT : Any> : Fragment() {
     container: ViewGroup?,
     savedInstanceState: Bundle?
   ): WorkflowLayout {
-    return WorkflowLayout(inflater.context)
-  }
-
-  override fun onViewStateRestored(savedInstanceState: Bundle?) {
-    super.onViewStateRestored(savedInstanceState)
-
-    _runner = WorkflowRunner.startWorkflow(this, ::onCreateWorkflow)
-
-    (view as WorkflowLayout).start(runner.renderings, viewEnvironment)
+    return WorkflowLayout(inflater.context).also { newView ->
+      _runner = WorkflowRunner.startWorkflow(this, ::onCreateWorkflow)
+      newView.start(runner.renderings, viewEnvironment)
+    }
```

Comment:
Could you add a comment explaining why this is necessary?

Reply:
```
    // https://github.com/square/workflow-kotlin/issues/14
    // We're careful to start up the workflow runtime before the view is attached,
    // since that's what LayoutRunner promises. When we're sloppy about that, we
    // break things like Jetpack Navigation and nested fragments.
```

Reply:
Awesome, thanks!